src/Microsoft.Azure.WebJobs.Host/Scale/IScaleManager.cs

```diff
@@ -1,20 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System.Threading.Tasks;
-
-namespace Microsoft.Azure.WebJobs.Host.Scale
-{
-    /// <summary>
-    /// Provides scale status.
-    /// </summary>
-    public interface IScaleManager
```

Comment:
Issue was that this interface had a general name about scale management, but only exposed a single status API. Would have renamed the interface to reduce its scope to something like IHostScaleMonitor, or IHostScaleStatusProvider. However, I don't think we need an interface at all. Like we do for ConcurrencyManager, we can expose the service type directly which allows us to add things in the future.